### PR TITLE
Fixed some stuff

### PR DESCRIPTION
**Pull request by Cybelyrium**
- Added link to the troubleshooting page in Finalizing Setup + GodMode9 Usage
- Fixed a h2wtest link and moved some sections that were misplaced

### CODE REVIEW

File: _pages/en_US/troubleshooting.txt [hunk 1]
```diff
@@ -467,7 +467,7 @@ Make sure you have at least 1.3GB available in your SD card. If you don't have e
 1. Copy the `Nintendo 3DS` folder from your computer to the root of your SD card
 1. Delete the `Nintendo 3DS` folder from your computer
 
-If you have enough space on your SD card, your SD might be corrupted or faulty. Check your SD card for any errors by following the guide according to your computer's operating system: [Windows](h2testw-(windows)), [Linux](f3-(linux)), [macOS](f3xswift-(mac)).
+If you have enough space on your SD card, your SD might be corrupted or faulty. Check your SD card for any errors by following the guide according to your system: [Windows](h2testw-(windows), [Linux](f3-(linux)), [macOS](f3xswift-(mac)).
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:470]: change was reverted

File: _pages/en_US/troubleshooting.txt [hunk 2]
```diff
@@ -564,10 +550,16 @@ ARM11 exception handlers are disabled, or custom firmware is not installed. Try
 <details>{{ compat | markdownify }}</details>
 
 {% capture compat %}
-<summary><u>Exception screen when booting/loading an application</u></summary>
+<summary><u>My device does not show any of my installed apps on the HOME Menu</u></summary>
 
-Look for your exception screen in [this page](https://wiki.hacks.guide/wiki/3DS:Error_screens/Luma3DS_exception_screen).
-If you weren't able to find your error or the instructions didn't work, join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) for further assistance.
+This could be caused by various reasons, but mostly because your SD card is not being read by the system.
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:555]: mostly -> most likely

Comment (lilyuwuu) [reply _pages/en_US/troubleshooting.txt:555]: system -> device (referring to 3DS)

File: _pages/en_US/troubleshooting.txt [hunk 3]
```diff
@@ -564,10 +550,16 @@ ARM11 exception handlers are disabled, or custom firmware is not installed. Try
...
+You can check if your SD is not being read by holding SELECT on boot and checking the yellow text on the bottom screen; if it says "Booted from CTRNAND via B9S", then your system is booting from the internal memory and not from the SD card.
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:556]: not being read -> being read (negation doesn't particularly matter but it flows better when not negated)

File: _pages/en_US/troubleshooting.txt [hunk 4]
```diff
@@ -564,10 +550,16 @@ ARM11 exception handlers are disabled, or custom firmware is not installed. Try
...
+Attempt the steps below, which are listed from easiest to hardest:
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:557]: If this is the case, attempt

File: _pages/en_US/troubleshooting.txt [hunk 5]
```diff
@@ -564,10 +550,16 @@ ARM11 exception handlers are disabled, or custom firmware is not installed. Try
...
+1. Power off your device, remove the SD card, re-insert it, then power on your device
+1. Power off your device, remove the SD card, insert it on your PC, download the latest release of [Luma3DS](https://github.com/LumaTeam/Luma3DS/releases/latest), extract `boot.firm` from the `Luma3DS.zip` and place it on the root of your SD card (replace it if it already exists)
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:559]: replacing any existing file

File: _pages/en_US/troubleshooting.txt [hunk 6]
```diff
@@ -564,10 +550,16 @@ ARM11 exception handlers are disabled, or custom firmware is not installed. Try
...
+1. Power off your device, remove the SD card, insert it on your PC and reformat your SD card (this will wipe your SD card data) according to your computer's operating system: [Windows](formatting-sd-(windows)), [macOS](formatting-sd-(mac)), [Linux](formatting-sd-(linux))
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:560]: sd card wipe note should go after formatting SD Linux - can go before but is a little clunky. Might also be good to bold it

File: _pages/en_US/troubleshooting.txt (hunk 5 above)
Comment (lilyuwuu) [reply _pages/en_US/troubleshooting.txt:559]: Also a bit of a run on instruction set but not sure of a better way to convey what is effectively six things into one bullet point

File: _pages/en_US/troubleshooting.txt [hunk 7]
```diff
@@ -564,10 +550,16 @@ ARM11 exception handlers are disabled, or custom firmware is not installed. Try
 <details>{{ compat | markdownify }}</details>
 
 {% capture compat %}
-<summary><u>Exception screen when booting/loading an application</u></summary>
+<summary><u>My device does not show any of my installed apps on the HOME Menu</u></summary>
```
Comment (lilyuwuu) [inline _pages/en_US/troubleshooting.txt:553]: Maybe "HOME Menu is missing installed applications"?

File: _pages/en_US/troubleshooting.txt (hunk 1 above)
Comment (lifehackerhansol) [reply _pages/en_US/troubleshooting.txt]: still reverted, see the h2testw link bracket

**Pull request merged**